src/SAWScript/Interpreter.hs

```diff
@@ -1314,7 +1307,7 @@ primitives = Map.fromList
     ]
 
   , prim "abc"                 "ProofScript ()"
```

Comment:
I looks like you removed `abc` from pretty much all of the example scripts and such. Should this command be `Deprecated` instead?

Reply:
I'm still debating that, honestly. I've tried making `abc` map to `w4_abc_aiger`, `w4_abc_smtlib2`, and `w4_abc_verilog`, and there are always cases where at least one of the benchmarks that worked well with the old `abc` doesn't work well with that choice (though all benchmarks seem to work well with at least one prover, either a mode of ABC or Yices). So, does having `abc` still exist for backward compatibility make sense if it will sometimes work equally well and sometimes not? It's hard to say.

Reply:
Any interest in having the "best" solver for each example discovered automatically eventually?

Reply:
Any interest in having the "best" solver for each example discovered automatically eventually? @kquick could be a good future use of the dependency reasoning automation you're working on.

Reply:
Yeah, Cryptol has `any` and `all` solvers and we might want to adopt a similar approach in SAW.

And there are still a bunch of uses of `abc` in the integration tests, even though I changed a few. Also, many of the external verifications (s2n, AWS-LC, BLS, etc.) use it. So, given that it does still exist in some form, I'm inclined to leave it.